api_payments.yaml

```diff
@@ -0,0 +1,286 @@
+swagger: "2.0"
```

Comment:
I would just call the file `api_pagopa` since it's not really a "payments" API (the payments API is the one exposed by the Wallet)